Dockerfile-ppc64le

```diff
@@ -0,0 +1,8 @@
+FROM ppc64le/docker:17.12.0-dind
+MAINTAINER David Wilder <wilder@ibm.com>
+
+# Install iptables, ip6tables, iproute2, and perform glibc install as per:
```

Comment:
This mentions glibc but it doesn't look like you install that.  IIRC, I think the glibc package isn't required for Calico, it comes from some other uses that the Tigera team have for the image so I think it's OK to remove it from here.

Reply:
ok,  I drop the entire comment, it is not needed.  Thanks for the review.